README-ta.md

```diff
@@ -0,0 +1,1830 @@
+
+### கணினி வடிவமைப்பு நேர்காணலுக்கான தயாரிப்பு
+
+குறியீட்டு நேர்காணல்களுடன் கூடுதலாக, கணினி வடிவமைப்பு என்பது பல தொழில்நுட்ப நிறுவனங்களில் **தொழில்நுட்ப நேர்காணல் செயல்முறை**யின் **தேவையான கூறு** ஆகும்.
+
+**பொதுவான சிஸ்டம் வடிவமைப்பு நேர்காணல் கேள்விகளைப் பயிற்சி செய்யவும்** மற்றும் **உங்கள் முடிவுகளை **மாதிரி தீர்வுகளுடன் ஒப்பிடவும்**: விவாதங்கள், குறியீடு மற்றும் வரைபடங்கள்.
+
+நேர்முகத் தயாரிப்புக்கான கூடுதல் தலைப்புகள்:
+* [Study guide](#study-guide)
+* [How to approach a system design interview question](#how-to-approach-a-system-design-interview-question)
+* [System design interview questions, **with solutions**](#system-design-interview-questions-with-solutions)
+* [Object-oriented design interview questions, **with solutions**](#object-oriented-design-interview-questions-with-solutions)
+* [Additional system design interview questions](#additional-system-design-interview-questions)
+
+## அங்கி ஃபிளாஷ் கார்டுகள்
+<p align="center">
+  <img src="images/zdCAkB3.png">
+  <br/>
+</p>
+வழங்கப்பட்டுள்ள [Anki flashcard decks](https://apps.ankiweb.net/) முக்கிய சிஸ்டம் வடிவமைப்புக் கருத்துகளைத் தக்கவைக்க உங்களுக்கு உதவ, இடைவெளியில் திரும்பத் திரும்பப் பயன்படுத்துகிறது.
+* [System design deck](https://github.com/donnemartin/system-design-primer/tree/master/resources/flash_cards/System%20Design.apkg)
+* [System design exercises deck](https://github.com/donnemartin/system-design-primer/tree/master/resources/flash_cards/System%20Design%20Exercises.apkg)
+* [Object oriented design exercises deck](https://github.com/donnemartin/system-design-primer/tree/master/resources/flash_cards/OO%20Design.apkg)
+
+பயணத்தின் போது பயன்படுத்த சிறந்தது.
+
+### குறியீட்டு ஆதாரம்: ஊடாடும் குறியீட்டு சவால்கள்
+
+[**குறியீட்டு நேர்காணலுக்கு**](https://github.com/donnemartin/interactive-coding-challenges) தயாராவதற்கு உங்களுக்கு உதவ ஆதாரங்களைத் தேடுகிறீர்களா?
+<p align="center">
+  <img src="images/b4YtAEN.png">
+  <br/>
+</p>
+சகோதரி ரெப்போவைப் பாருங்கள் [**ஊடாடும் குறியீட்டு சவால்கள்**](https://github.com/donnemartin/interactive-coding-challenges), இதில் கூடுதல் Anki டெக் உள்ளது:
+. [Coding deck](https://github.com/donnemartin/interactive-coding-challenges/tree/master/anki_cards/Coding.apkg)
+## பங்களிக்கிறது
+> சமூகத்திலிருந்து கற்றுக்கொள்ளுங்கள்.
+
+உதவிக்கு இழுக்கும் கோரிக்கைகளைச் சமர்ப்பிக்க தயங்க வேண்டாம்:
+
+* பிழைகளை சரிசெய்யவும்
+* பிரிவுகளை மேம்படுத்தவும்
+* புதிய பிரிவுகளைச் சேர்க்கவும்
+* [Translate](https://github.com/donnemartin/system-design-primer/issues/28)
+சில மெருகூட்டல் தேவைப்படும் உள்ளடக்கம் [வளர்ச்சியில்] (#குறை வளர்ச்சி) வைக்கப்படுகிறது.
+
+[பங்களிப்பு வழிகாட்டுதல்கள்] மதிப்பாய்வு செய்யவும்(CONTRIBUTING.md).
+## Index of system design topics
+
+> Summaries of various system design topics, including pros and cons.  **Everything is a trade-off**.
+>
+> Each section contains links to more in-depth resources.
+
+<p align="center">
+  <img src="images/jrUBAF7.png">
+  <br/>
+</p>
+* [System design topics: start here](#system-design-topics-start-here)
```

Comment:
**Why you have not translated the menu ???**

Reply:
Yea just saw it now will fix it working on it ;-)

Reply:
make this simple